app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/FractionInteractionViewModel.kt

```diff
@@ -47,8 +48,12 @@ class FractionInteractionViewModel private constructor(
           )
         }
       }
+    if (rawUserAnswer != null) {
+      answerText = rawUserAnswer.fraction
+    }
     errorMessage.addOnPropertyChangedCallback(callback)
     isAnswerAvailable.addOnPropertyChangedCallback(callback)
+    checkPendingAnswerError(AnswerErrorCategory.REAL_TIME)
```

Comment:
Are there tests to verify that the error is properly initialized with the view model's initialization (e.g. that would fail if this line were removed in the future)?

Reply:
In tests, I am also checking that whether error message is getting displayed or not in screen rotation.